src/reducers/Artisans.js

```diff
@@ -1,12 +1,23 @@
 export default function Artisans(state = [], action) {
-  console.log("Artisans reducing action " + action.type)
+  
```

Comment:
probably dont want to remove this comment. Theyre pretty helpful for debugging reducers